src/ui.js

```diff
@@ -302,6 +302,52 @@ const ui = {
     //console.log("un_hookup_ui", elem.id)
   },
 
+  injectSvgNode: (node, targetNode, width, height = width) => {
+    node.setAttribute('x', 0)
+    node.setAttribute('y', 0)
+    const originalWidth = node.getAttribute('width')
+    const originalHeight = node.getAttribute('height')
+
+    targetNode.setAttribute('width', width)
+    targetNode.setAttribute('height', height)
+    targetNode.setAttribute('viewBox', `0 0 ${originalWidth} ${originalHeight}`)
+    targetNode.appendChild(node)
+  },
+
+  generateActivityItemFromTemplate: (playerAction, beforeNode, afterNode) => {
+    const templateActivityItem = document.querySelector('#template_activity_item')
+
+    templateActivityItem.content.querySelector('.activity_item_timestamp').innerText = new Date().toLocaleTimeString()
```

Comment:
do you need `.content` before `.querySelector`?  Or is it some kind of best-practice?  I've never seen that before

Reply:
Use of `.content` is based on the examples in the article you sent to me. 

![image](https://user-images.githubusercontent.com/6127892/106676973-b86da980-656c-11eb-9016-f36a04f459ce.png)
https://www.html5rocks.com/en/tutorials/webcomponents/template/

I tested, and removing `.content` causes errors.

Reply:
good enough for me then